src/source/view/BaseScreen.bs

```diff
@@ -85,6 +85,10 @@ namespace mv
       replaceNavControllerScreen(targetScreen, animated)
     end function
 
+    protected override function cancelPendingSelection(selection as mc.types.node)
+      super.cancelPendingSelection(selection)
```

Comment:
I don't understand this. what is this doing? This is identical to not having this function.. all you're doing is calling the super.